app/decorators/numismatics/accession_decorator.rb

```diff
@@ -75,9 +75,9 @@ def formatted_date
     end
 
     def parsed_date
-      if date.to_s.include?("/")
+      if date.to_s.include?("-")
         # It's an american hand-entered date, parse it as such
-        Time.zone.strptime(date, "%m/%d/%Y")
+        Time.zone.strptime(date.to_s, "%Y-%m-%d")
```

Comment:
Without adding .to_s, it throws "TypeError: no implicit conversion of DateTime into String" and does not actually use the specified format in parsed date. (This previously caused a test to fail by putting the original date with the time attached in formatted output.)